RecoLocalCalo/EcalRecProducers/python/ecalRecHitGPU_cfi.py

```diff
@@ -0,0 +1,132 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoLocalCalo.EcalRecAlgos.ecalCleaningAlgo import cleaningAlgoConfig 
+
+# rechit producer
+ecalRecHitGPU = cms.EDProducer("EcalRecHitProducerGPU",
```

Comment:
can you simplify this based on (or fully replace by) the cfi file produced by `fillDescriptions()` ?

Reply:
I could remove it.
It is still useful for debug and development, for possible extensions (e.g. rechit recovery) that currently I have not migrated to gpu.
If it's not a problem, I prefer to leave it for this reason.

Reply:
OK